features/users/user_delete.feature

```diff
@@ -117,3 +117,14 @@ Scenario: Delete a user who has coauthored a work
       And a user account should not exist for "testuser"
     When I go to orphan_account's series page
     Then I should see "Epic"
+
+  Scenario: Deleting user updates kudos fragment
```

Comment:
```suggestion
  Scenario: Deleting user updates kudos fragment cache
```
would be a bit clearer for what is getting tested. Same for the other test.

Reply:
I think these tests are misleading, tbh. The cache fragment isn't actually getting expired by the user deletion or rename; it just gets expired periodically. 

We'd rather the tests go in kudos.feature and be rephrased to something like "Kudos cache expires periodically to ensure deleted users are removed and renamed users are updated." Both the rename and the deletion can go in the same scenario.

Reply:
Moved the test as @sarken suggested!